vendor/knative.dev/hack/library.sh

```diff
@@ -490,24 +490,37 @@ function start_latest_eventing_sugar_controller() {
 function run_go_tool() {
   local tool=$2
   local install_failed=0
-  if [[ -z "$(which ${tool})" ]]; then
-    local action=get
-    [[ $1 =~ ^[\./].* ]] && action=install
-    # Avoid running `go get` from root dir of the repository, as it can change go.sum and go.mod files.
-    # See discussions in https://github.com/golang/go/issues/27643.
-    if [[ ${action} == "get" && $(pwd) == "${REPO_ROOT_DIR}" ]]; then
-      local temp_dir="$(mktemp -d)"
-      # Swallow the output as we are returning the stdout in the end.
-      pushd "${temp_dir}" > /dev/null 2>&1
-      GOFLAGS="" go ${action} "$1" || install_failed=1
-      popd > /dev/null 2>&1
-    else
-      GOFLAGS="" go ${action} "$1" || install_failed=1
+  local run=$1
+
+  if [[ "$(basename $1)" != "$2" ]]; then
```

Comment:
Is there a reason that you use $1 & $2 here, instead of $run & $tool?

Reply:
I didn't make that change. This is updated by running `./hack/update-deps.sh --upgrade` and ` ./hack/update-codegen.sh `, related PR merged in vendor repo is https://github.com/knative/hack/pull/38

Reply:
oh right. I didn't check it was in vendor/. nvm.